include/z64audio.h

```diff
@@ -618,13 +231,13 @@ typedef struct {
     /* 0x4368 */ u8 sampleFontLoadStatus[0x30];
     /* 0x4398 */ u8 fontLoadStatus[0x30];
     /* 0x43C8 */ u8 seqLoadStatus[0x80];
-    /* 0x4448 */ volatile u8 resetStatus;
+    /* 0x4448 */ vu8 resetStatus;
     /* 0x4449 */ u8 specId;
     /* 0x444C */ s32 audioResetFadeOutFramesLeft;
     /* 0x4450 */ f32* adsrDecayTable; // A table on the audio heap that stores decay rates used for ADSR
     /* 0x4454 */ u8* audioHeap;
     /* 0x4458 */ size_t audioHeapSize;
-    /* 0x445C */ Note* notes;
+    /* 0x445C */ struct Note* notes;
```

Comment:
```suggestion
    /* 0x445C */ struct Note* notes; // dynamically sized based on gAudioCtx.numNotes
```
When I see one of these non-static size I always want to know more, thoughts?

Reply:
lgtm